README.md

```diff
@@ -2,15 +2,15 @@
 
 A Github Action to automatically bump and tag master, on merge, with the latest SemVer formatted version.
 
-[![Build Status](https://github.com/anothrNick/github-tag-action/workflows/Bump%20version/badge.svg)](https://github.com/anothrNick/github-tag-action/workflows/Bump%20version/badge.svg)
-[![Stable Version](https://img.shields.io/github/v/tag/anothrNick/github-tag-action)](https://img.shields.io/github/v/tag/anothrNick/github-tag-action)
-[![Latest Release](https://img.shields.io/github/v/release/anothrNick/github-tag-action?color=%233D9970)](https://img.shields.io/github/v/release/anothrNick/github-tag-action?color=%233D9970)
+[![Build Status](https://github.com/geoadmin/github-tag-action/workflows/Bump%20version/badge.svg)](https://github.com/geoadmin/github-tag-action/workflows/Bump%20version/badge.svg)
+[![Stable Version](https://img.shields.io/github/v/tag/geoadmin/github-tag-action)](https://img.shields.io/github/v/tag/geoadmin/github-tag-action)
+[![Latest Release](https://img.shields.io/github/v/release/geoadmin/github-tag-action?color=%233D9970)](https://img.shields.io/github/v/release/geoadmin/github-tag-action?color=%233D9970)
 
```

Comment:
?

Reply:
I've noticed this is a recurring issue in PRs. Would it be possible to have a bot that reminds people to rename these values (in main.yml and README.md) when a PR is opened?